modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -41,8 +41,32 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * GCP: `LoadBalancerService` (with External scope)
 * Bare metal: `NodePortService`
 * Other: `HostNetwork`
++
+[NOTE]
+====
+`HostNetwork` has a `hostNetwork` field with the following default values for the optional binding ports: `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`.
+With the binding ports, you can deploy multiple Ingress Controllers on the same node for the `HostNetwork` strategy.
+
+.Example
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
```

Comment:
@Amrita42, this is indented wrong; Should be flush left. Refer to almost any other YAML in our documentation for how this is done.